npc/re/kafras/Zonda_Teleportation_Services.txt

```diff
@@ -0,0 +1,95 @@
+//===== rAthena Script ======================================= 
+//= Zonda Teleportation Services
+//===== Changelogs: ==========================================
+//= 1.0 First Version. [JohnnyPlayy]
+//===== Description: ========================================= 
+// Texts from this script were translated according to KRO
+//============================================================
+
+-	script	ZondaTeleportationServices	HIDDEN_NPC,{
+	cutin "zonda_do01", 2;
+	mes "[Zonda Employee]";
+	mes "Welcome.";
+	mes "Our team is always working to exceed your expectations regarding quality of service.";
+	mes "Please tell me your destiny?";
+	next;
+	switch( select( "North Bifrost", "Eclage", "Splendide", "Midgard Expedition Camp", "Manuk", "Outskirts of Kamidal Mountain", "Cancel.") ) {
+	case 1:
+		callsub( S_Zone, "North Bifrost", 17000,850, 7213 ); // Teleport value not confirmed
+		warp "bif_fild02",160,230; // Unverified Coordinates
+		end;
+	case 2:
+		callsub( S_Zone, "Eclage", 17000,850, 11310 );
```

Comment:
The quest requirement is probably
```suggestion
		callsub( S_Zone, "Eclage", 17000,850, 11311 );
```
https://github.com/rathena/rathena/blob/468d6f4d96c647e71d776e4f2a80161c085f9039/npc/re/quests/quests_eclage.txt#L164